manifest.yaml

```diff
@@ -117,6 +117,18 @@ postprocess:
      grep -q '# RHEL-only: Disable /tmp on tmpfs' /usr/lib/systemd/system/basic.target
      echo '# RHCOS-only: we follow the Fedora/upstream default' >> /usr/lib/systemd/system/basic.target
      echo 'Wants=tmp.mount' >> /usr/lib/systemd/system/basic.target
+
+     # TEMPORARY: Remove `ipmi-si` in /usr/lib/modules-load.d/fwupd-redfish.conf until fix is landed to RHEL 8.6
+     # Fixed version is fwupd-1.7.4-1.el8
+     # See https://github.com/openshift/os/issues/749
+     #     https://bugzilla.redhat.com/show_bug.cgi?id=2037294
+     #     https://github.com/fwupd/fwupd/pull/4162
+     . /etc/os-release
+     if [ "$(uname -m)" == "x86_64" ]; then
+     [ "${ID}" == "rhel" ] && [ ${VERSION_ID//.} -ne 86 ] && exit 0
```

Comment:
I don't think we need this one if we do the one below

Reply:
For this, I just do not want to make sure this does not affect rhel8.5
Check on rhel8.5, no found /usr/lib/modules-load.d/fwupd-redfish.conf

Reply:
Maybe I should check the file before grep